apps/meteor/ee/app/license/server/getStatistics.ts

```diff
@@ -81,6 +83,24 @@ async function getEEStatistics(): Promise<EEOnlyStats | undefined> {
 			}),
 	);
 
+	// Number of PDF transcript requested
+	statsPms.push(
+		LivechatRooms.find({ pdfTranscriptRequested: { $exists: true } })
```

Comment:
`pdfTranscriptRequested` is not indexed, this query can cause issue since the rooms collection can be somewhat big.

since you're doing different queries for PDF transcripts statuses, what about having a single new "pdf status" field, creating a sparse index for it and using it for both queries?

Reply:
I'll keep them separated cause one signifies that a transcript was requested, but not yet completed, while the other `pdfFileId` signifies transcript was done.

On the index, yeah, i have them on my db but not on the code so maybe they got ripped by some conflict :sadblob: